src/core_plugins/kibana/common/tutorials/filebeat_instructions.js

```diff
@@ -102,8 +102,8 @@ export const createFilebeatInstructions = context => ({
         }
       }),
       commands: [
-        'PS > cd C:\\Program Files\\Filebeat',
-        'PS C:\\Program Files\\Filebeat> .\\install-service-filebeat.ps1',
+        'cd C:\\Program Files\\Filebeat',
```

Comment:
Do we want there to be quotes here like below?
`cd "C:\\Program Files\\Filebeat"`

Reply:
Good catch!